src/__integration_tests__/getCollection.ts

```diff
@@ -0,0 +1,47 @@
+import { assert } from "chai";
+import { ethers } from "ethers";
+import { suite, test } from "mocha";
+import Web3 from "web3";
+import { Network } from "wyvern-js/lib/types";
+import {
+  ALCHEMY_API_KEY,
+  MAINNET_API_KEY,
+  WALLET_PRIV_KEY,
+} from "../__tests__/constants";
+import { OpenSeaSDK } from "../sdk";
+import { SafelistStatus } from "../types";
+
+const webProvider = new Web3.providers.HttpProvider(
+  `https://eth-mainnet.g.alchemy.com/v2/${ALCHEMY_API_KEY}`
+);
+
+const rpcProvider = new ethers.providers.JsonRpcProvider(
+  `https://eth-mainnet.g.alchemy.com/v2/${ALCHEMY_API_KEY}`
+);
+
+const wallet = new ethers.Wallet(WALLET_PRIV_KEY ?? "", rpcProvider);
+
+const sdk = new OpenSeaSDK(
```

Comment:
maybe move this to a shared file for integ tests

Reply:
(all of the initialization logic)

Reply:
Done